### ISSUE TITLE
When processing folders total file count displayed in status bar is 0

### ISSUE THREAD

**Issue by paul1956** [paul1956/CSharpToVB#18]
This is a regression from before making converter Async @jnm2

**Comment by jnm2** [comment]
I see my mistake. When I was looking for why `TotalFilesToProcess` was `ByRef`, I missed the fact that it calls itself recursively. Fixing!

**Comment by jnm2** [comment]
Actually never mind- I still don't see anywhere that TotalFilesToProcess is changed once set.

**Comment by jnm2** [comment]
@paul1956 Do you have specific steps I can take? I can't reproduce this problem. This is what I get with Convert > Convert Folder:

![image](https://user-images.githubusercontent.com/8040367/76714355-ed76d080-66fc-11ea-8f9c-5f340d288894.png)

**Comment by jnm2** [comment]
Another try:

![image](https://user-images.githubusercontent.com/8040367/76714422-40e91e80-66fd-11ea-8922-976436f72fc7.png)

This is using commit https://github.com/paul1956/CSharpToVB/commit/945b37f8e04d3a4c5d007835da3e9fe2f7948689 which is currently the tip of master.

**Comment by paul1956** [comment]
I think I fixed it it the last commit, but the total count may be off now in some cases.

**Comment by paul1956** [comment]
Actually it is an issue where the processing code skips some files (auto generates, Ignored...) in the original code total is decremented. I am testing now.

**Comment by paul1956** [comment]
![image](https://user-images.githubusercontent.com/12789369/76721411-45153c00-66fd-11ea-9918-c707d54cf665.png)
This is when complete, the total is not being decremented.

**Comment by jnm2** [comment]
@paul1956 I still don't understand, I'm sorry. Where in the codebase before or after my PRs was the total being decremented? I can't find it even before my first PR. I see only one decrement in the codebase and it is not decrementing the TotalFilesToProcess variable which is what drives that message.

I tried running commit e4209a6df6afa689e5e3aef46805f9642c3eead5 which is the commit before my first PR merged. I got this:
![image](https://user-images.githubusercontent.com/8040367/76761241-7a557480-6765-11ea-8085-43de59bcb30c.png)
 
So I'm pretty sure this is *not* a regression from making it async but rather an existing underlying problem.
